Samples/word-citation-management/src/taskpane/taskpane.html

```diff
@@ -0,0 +1,77 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. -->
+<!-- This file shows how to design a first-run page that provides a welcome screen to the user about the features of the add-in. -->
+
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Citation Manager Task Pane Add-in</title>
+
+    <!-- jQuery -->
+    <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.6.0.js" type="text/javascript"></script>
+
+    <!-- Office JavaScript API -->
+    <script type="text/javascript" src="https://appsforoffice.microsoft.com/lib/1.1/hosted/office.js"></script>
+
+    <!-- For more information on Fluent UI, visit https://developer.microsoft.com/fluentui#/. -->
+    <link rel="stylesheet" href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/11.0.0/css/fabric.min.css"/>
+
+    <!-- Template styles -->
+    <link href="taskpane.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body class="ms-font-m ms-welcome ms-Fabric ms-bgColor-white">
+    <header id="content-header">
+        <div class="ms-Grid ms-bgColor-neutralPrimary">
+            <div class="ms-Grid-row">
+                <div class="padding ms-Grid-col ms-u-sm12 ms-u-md12 ms-u-lg12">
+                    <div class="ms-font-xl ms-fontColor-white ms-fontWeight-semibold">Citation management</div>
+                </div>
+            </div>
+        </div>
+    </header>
+
+    <main id="app-body" class="ms-welcome__main" style="display: none;">
+        <h2 class="ms-font-l">Bibliography sources</h2>
+        Select the bibliography file you'd like to use.
+        <form>
+            <input type="file" id="bib-file" name="bib-file" accept=".bib">
+        </form>
+        <div id="references-section" style="display: none;">
+            <h2 class="ms-font-xl">References</h2>
+            <p><strong>Important</strong>: In the document, place the cursor where you want to insert the selected reference.</p>
```

Comment:
I'm not sure you need this instruction. The usage feels intuitive enough (where else would the reference go?) You also have the instructions in the readme, so having the add-in focus on functionality is fine.